master/internal/trial_workload_sequencer.go

```diff
@@ -110,78 +109,82 @@ func (s *trialWorkloadSequencer) SetTrialID(trialID int) {
 }
 
 // WorkloadCompleted receives the searcher.CompletedMessage and updates the internal state of the
-// trialWorkloadSequencer accordingly. It is the only method that should update the state.
+// trialWorkloadSequencer accordingly. It and WorkloadFailed are the only methods that should update the state.
 // It snapshots this state whenever we've just completed a checkpoint, and should be the only method
 // to alter the snapshot, too.
 func (s *trialWorkloadSequencer) WorkloadCompleted(
```

Comment:
There's a lot i would cleanup in here now if it were necessary, but it's totally not, since this is moving to python.